.github/scripts/build.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+export DEB_SIGN_PROGRAM="gpg --pinentry-mode loopback --passphrase-file ${HOME}/signing.passphrase"
+export TARBALL_BASE_URL="https://pypi.python.org/packages/source"
```

Comment:
This is a long-deprecated address. It should be `pypi.org`.

Reply:
thank you for catching this, I will update this in a future PR